framework/src/modules/chain/modules/rounds.js

```diff
@@ -62,6 +64,128 @@ class Rounds {
 	}
 }
 
+__private.updateSendersRoundInformationWithAmountForTransactions = function(
```

Comment:
`updateSendersRoundInformationWithAmount` is good enough to explain. We may ned up function names with line character limit.